examples/node/package.json

```diff
@@ -4,7 +4,8 @@
   "type": "commonjs",
   "scripts": {
     "build": "rsbuild build",
-    "test": "rstest run"
+    "test": "rstest run",
+    "test:watch": "rstest"
```

Comment:
why not use `rstest watch` command?

<img width="604" alt="image" src="https://github.com/user-attachments/assets/04bc74b7-d99f-4b43-aa8e-b5a283b9d75f" />

Reply:
yeah, fixed!